File: Modul-2/Soal 4/PRAK204-2410817210014-ZAVIER GHANI.py
pi = 3.14

# Test Case 1
print("Test Case 1")
r1 = float(input("Masukkan jari-jari dan tinggi (dipisah spasi): ").split()[0])
h1 = float(input().strip())  

volume1 = pi * r1 * r1 * h1
luas1 = 2 * pi * r1 * (r1 + h1)
keliling1 = 2 * pi * r1

print(f"Volume = {volume1:.2f}")
print(f"Luas = {luas1:.2f}")
print(f"Keliling = {keliling1:.2f}")

print("\n") 

# Test Case 2
print("Test Case 2")
r2, h2 = map(float, input("Masukkan jari-jari dan tinggi (dipisah spasi): ").split())

volume2 = pi * r2 * r2 * h2
luas2 = 2 * pi * r2 * (r2 + h2)
keliling2 = 2 * pi * r2

print(f"Volume = {volume2:.2f}")
print(f"Luas = {luas2:.2f}")
print(f"Keliling = {keliling2:.2f}")

File: Modul-2/Soal 4/PRAK204-2410817210014-ZAVIERGHANI.c
#include <stdio.h>

int main() {
    // Test Case 1
    float r1, h1;

    printf("Test Case 1\n");
    printf("Masukkan jari-jari dan tinggi (dipisah spasi): ");
    scanf("%f %f", &r1, &h1);

    float volume1 = 3.14 * r1 * r1 * h1; 
    float luas1 = 2 * 3.14 * r1 * (r1 + h1); 
    float keliling1 = 2 * 3.14 * r1; 

    printf("Volume = %.2f\n", volume1);
    printf("Luas = %.2f\n", luas1);
    printf("Keliling = %.2f\n", keliling1);

    printf("\n"); 

    // Test Case 2
    float r2, h2;

    printf("Test Case 2\n");
    printf("Masukkan jari-jari dan tinggi (dipisah spasi): ");
    scanf("%f %f", &r2, &h2);

    float volume2 = 3.14 * r2 * r2 * h2;
    float luas2 = 2 * 3.14 * r2 * (r2 + h2);
    float keliling2 = 2 * 3.14 * r2; 

    printf("Volume = %.2f\n", volume2);
    printf("Luas = %.2f\n", luas2);
    printf("Keliling = %.2f\n", keliling2);

    return 0;
}
